Simplenote/src/main/res/drawable/splash_background.xml

```diff
@@ -0,0 +1,9 @@
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android"
+    android:opacity="opaque">
+    <item android:drawable="@color/blue"/>
+    <item
+        android:width="72dp"
+        android:height="72dp"
```

Comment:
```suggestion
        android:height="@dimen/icon_empty"
```

Let's use the `icon_empty` dimension resource for the height rather than hard-coding `72dp`.   The `icon_empty` value is also `72dp` and that's what @SylvesterWilmott originally suggested.

> I think the logo looks quite large on screen, can we try 72dp which matches with our empty state icon sizes?